Diagnostic/diagnostic.py

```diff
@@ -499,7 +498,7 @@ def start_mdsd(configurator):
                                                        g_mdsd_bin_path, xml_file, g_ext_dir)
     config_validate_cmd_status, config_validate_cmd_msg = RunGetOutput(config_validate_cmd)
     if config_validate_cmd_status is not 0:
-        # Invalid config. Log error and report success.
+        # Invalid config, return error.
```

Comment:
There doesn't seem to be a code change here. Was the previous comment wrong?

Reply:
It wasn't, but I modified the function `log_and_report_invalid_mdsd_cfg()` in misc_helpers.py to return an error rather than report success.